packages/tasks/src/local-apps.ts

```diff
@@ -389,6 +399,13 @@ export const LOCAL_APPS = {
 		displayOnModelPage: (model) => model.library_name === "diffusers" && model.pipeline_tag === "text-to-image",
 		deeplink: (model) => new URL(`https://models.invoke.ai/huggingface/${model.id}`),
 	},
+	ollama: {
+		prettyLabel: "Ollama",
+		docsUrl: "https://ollama.com",
```

Comment:
Probably have dedicated docs on the hub for this, but we can make a follow-up PR later on